docs/get-started.md

```diff
@@ -26,13 +26,41 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
 
 ### Set up server
 
-1. Start the server and the local storage service. In the root directory of Ark, run:
+These instructions assume that you are running Minio inside your cluster.
+
+1. Start the local storage service. In the root directory of Ark, run:
 
     ```bash
     kubectl apply -f examples/common/00-prereqs.yaml
-    kubectl apply -f examples/minio/
+    kubectl apply -f examples/minio/00-minio-deployment.yaml
     ```
 
+1.  Get the Minio URL:
+
+    - if you're running Minikube
+
+    ```shell
+    minikube service minio --namespace=heptio-ark --url
+    ```
+
+    - in any other environment
+
+       1. Get the value of the IP address or DNS name of any node in your cluster.
```

Comment:
Probably should note that this needs to be an external IP (i.e. routable from where the `ark` client is being run)